lib/activities/get_extent.js

```diff
@@ -1,22 +1,27 @@
 "use strict";
 
-var url = require("url"),
+var debug = require("debug"),
+    url = require("url"),
     util = require("util");
 
 var shell = require("./shell");
 
+var log = debug("swfr:shell");
+
 /**
  * Fetch the extent of a raster.
  *
  * @returns [upper left, lower right]
  */
 module.exports = function getExtent(uri, callback) {
-  uri = url.parse(uri);
-
   var args = [
-    util.format("http://s3.amazonaws.com/%s%s", uri.hostname, uri.pathname)
+    "-nofl", // In case this is a VRT with too many files to be caught in stdout
```

Comment:
It just means that `gdalinfo` doesn't output all of the VRTs. We're only using gdalinfo for grabbing the extent, so we don't ever need the file list. If the file list is huge, we get a std.out overflow error of sorts...shell can't grab all the output.